File: src/main/java/com/manifestors/shinrai/client/module/modules/movement/SilkFall.java
package com.manifestors.shinrai.client.module.modules.movement;


import com.manifestors.shinrai.client.module.Module;
import com.manifestors.shinrai.client.module.ModuleCategory;
import com.manifestors.shinrai.client.module.annotations.ModuleData;

@ModuleData(
        name = "SilkFall",
        description = "Block fall damage",
        category = ModuleCategory.MOVEMENT,
        alternatives = "NoFall"

)
public class SilkFall extends Module {
}

File: src/main/java/com/manifestors/shinrai/client/module/modules/movement/SilkFall.kt
package com.manifestors.shinrai.client.module.modules.movement


import com.manifestors.shinrai.client.module.Module
import com.manifestors.shinrai.client.module.ModuleCategory

object SilkFall : Module(
    name = "SilkFall",
    description = "Block fall damage",
    category = ModuleCategory.MOVEMENT,
    alternativeNames = arrayOf("NoFall")
)
